_data/nav.yml

```diff
@@ -4,27 +4,27 @@
     - subtitle: "generator-cf"
       subhref: "https://github.com/cfpb/generator-cf"
     - subtitle: "Compiled source"
-      subhref: "https://cfpb.github.io/capital-framework/releases/capital-framework-latest.zip"
+      subhref: "http://registry.npmjs.org/capital-framework/-/capital-framework-3.1.1.tgz"
```

Comment:
I just downloaded it and it does contain the `dist` directory and files... so, yes kinda sorta?